lib/util.js

```diff
@@ -49,18 +49,15 @@ let util = {
         let size = parseInt(stdout.toString().split(/\s+/)[0]);
         if (Number.isNaN(size)) size = 0;
 
-        if (size && error)
-        {
+        if (size && error) {
           logger.warn('Errors occurred getting size of "' + directory + '", size might be incorrectly reported...');
           logger.debug(error);
           resolve(size);
         }
-        else if (!error)
-        {
+        else if (!error) {
```

Comment:
There are still several places where you changed the braces style before that need to be changed back.

If you run `git diff 41bc11a04838fc9189927ea3ca33bee6817e18a8 HEAD`, it will show you all the places where you changed this.